Docs build failures with Sphinx 3

bodhi fails to build from source on Fedora with Sphinx 3 series.

Downstream bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1823739

Error:
Warning, treated as error:
/builddir/build/BUILD/bodhi-5.1.1/bodhi/server/models.py:docstring of bodhi.server.models.Bug.url:1:duplicate object description of bodhi.server.models.Bug.url, other instance in developer/docblocks/bodhi.server.models, use :noindex: for one of them
make: *** [Makefile:51: html] Error 2
make: Leaving directory '/builddir/build/BUILD/bodhi-5.1.1/docs'
error: Bad exit status from /var/tmp/rpm-tmp.A6849v (%build)

Sphinx 3 changelog: https://www.sphinx-doc.org/en/master/changes.html#release-3-0-0-released-apr-06-2020

--- a/bodhi/server/models.py
+++ b/bodhi/server/models.py
@@ -4377,8 +4377,6 @@ class Bug(Base):
         bug_id (int): The bug's id.
         title (str): The description of the bug.
         security (bool): True if the bug is marked as a security issue.
-        url (str): The URL for the bug. Inaccessible due to being overridden by the url
-            property (https://github.com/fedora-infra/bodhi/issues/1995).
         parent (bool): True if this is a parent tracker bug for release-specific bugs.
     """
 
@@ -4394,9 +4392,6 @@ class Bug(Base):
 
     # If we're dealing with a security bug
     security = Column(Boolean, default=False)
-
-    # Bug URL.  If None, then assume it's in Red Hat Bugzilla
-    url = Column('url', UnicodeText)
 
     # If this bug is a parent tracker bug for release-specific bugs
     parent = Column(Boolean, default=False)

